src/common.rs

```diff
@@ -3635,6 +3651,38 @@ impl fmt::Display for MacAddr {
     }
 }
 
+/// Ip networks address for network interface.
+///
+/// It is returned by [`NetworkData::ip_networks`][crate::NetworkData::ip_networks].
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, PartialOrd, Ord)]
+pub struct IpNetwork {
+    addr: IpAddr,
+    prefix: u8,
+}
+
+impl IpNetwork {
+    /// build new IpNetwork
+    pub fn new(addr: IpAddr, prefix: u8) -> Self {
```

Comment:
With the fields public, this constructor won't be needed anymore too.

Reply:
Now I removed this constructor.